sample/ios-app/sample-appUITests/MoreTests.swift

```diff
@@ -27,6 +27,7 @@ class MoreTests: XCTestCase {
         super.tearDown()
     }
 
+    // @Flowers
```

Comment:
Instead of modifying existing tests we should just add more tests with these tags